### ISSUE TITLE
Select delimiter value

### ISSUE THREAD

**Issue by AvidSeeker** [andmarti1424/sc-im#785]
Add support for semicolon-separated values, pipe-separated values, etc. (Like `cut -d`)

**Comment by andmarti1424** [comment]
Please take a look at the doc. Its in there.

**Comment by avidseeker** [comment]
Right. I found the `--txtdelim` option in `sc-im --help` (`sc-im -h` doesn't work).

I was checking `man sc-im` and it didn't contain any info about it.
